### DISCUSSION TITLE
Issue Updating VaultWarden - LXC

### DISCUSSION THREAD

**Discussion by millie75** [tteck/Proxmox#1473] [General]
When I click on Update I get the below

[ERROR] in line 60: exit code 1: while executing command VAULT=$(curl -s https://api.github.com/repos/dani-garcia/vaultwarden/releases/latest |
    grep "tag_name" |
    awk '{print substr($2, 2, length($2)-3) }')

**Comment by tteck** [comment]
Cannot replicate

```
root@server:~# curl -s https://api.github.com/repos/dani-garcia/vaultwarden/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }'
1.28.1
root@server:~# 
```
Typically, the error you've provided suggests the presence of DNS problems, or github api issues.

**Reply by millie75** [reply]
Thanks must have been github api as now working fine, thanks for quick response
